src/getHumanAge.test.js

```diff
@@ -8,6 +8,43 @@ describe(`Function 'getHumanAge':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(getHumanAge(8, 1)).toBeInstanceOf(Array);
+  });
+
+  it(`should return [0, 0] with ages < 15`, () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  it(`should return [1, 1] when ages [15; 23]`, () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
+  it(`should return [1, 1] when ages [15; 23]`, () => {
+    expect(getHumanAge(23, 23)).toEqual([1, 1]);
+  });
+
+  it(`should return [2, 2] when catAge is 24 
+  and dogAge is 24`, () => {
+    expect(getHumanAge(24, 24)).toEqual([2, 2]);
+  });
```

Comment:
Please make all your tests in one style. This is the best option, use it for all tests in the suite